bigbluebutton-html5/imports/ui/components/chat/chat-graphql/chat-message-list/page/chat-message/component.tsx

```
@@ -291,21 +291,59 @@ const ChatMessage = React.forwardRef<ChatMessageRef, ChatMessageProps>(({
       return messages;
     });
   }, [messageContentRef]);
+
+  const scrollEndFrameRef = React.useRef<number | void>();
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)